src/packages/client/src/__tests__/integration/errors/connection-limit/test.ts

```diff
@@ -23,10 +23,9 @@ describe('connection-limit', () => {
       clients.push(client)
     }
 
-    await Promise.all(clients.map(c => c.$connect()))
 
     try {
-      await Promise.all(clients.map(c => c.$queryRaw(`SELECT 1`)))
+      await Promise.all(clients.map(c => c.$connect()))
```

Comment:
Nice. I didn't think the error would be thrown on connect, thought there would have to be a query but I guess not :)